prime-angular-frontend/src/app/shared/components/enrollee/enrollee-review/enrollee-review.component.ts

```diff
@@ -88,6 +89,12 @@ export class EnrolleeReviewComponent {
       : [];
   }
 
+  public get jobs(): Job[] {
```

Comment:
Shouldn't use a getter for access keys on a public object

Reply:
I used it here to keep the same coding style as a getter was used for oboSites, changed now but oboSites is accessed through a getter while jobs is accessed through enrolment.

Reply:
Good to know, for sure I'll remove the getter for oboSites

Reply:
Fixed for `oboSites` and `careSettings`